src/main/java/org/openstreetmap/atlas/checks/base/BaseCheck.java

```diff
@@ -128,8 +128,8 @@ public BaseCheck(final Configuration configuration)
         }
         catch (final Exception oops)
         {
-            logger.error(String.format("%s failed on feature %s.", this.getCheckName(),
-                    object.getIdentifier()), oops);
+            logger.error(String.format("%s failed on feature %s (%s).", this.getCheckName(),
+                    object.getIdentifier(), object.getOsmIdentifier()), oops);
```

Comment:
Is there a reason for this change? It appears to be redundant as the atlas identifier already contains the OSM id.

Reply:
yes. for debugging purpose its very convenient to copy original OSM id instead of playing with sectioned id in cases where original ID ends with 0 or small.

Reply:
@Bentleysb this change was integrated in https://github.com/osmlab/atlas-checks/pull/305/files